File: CSharp-Two/StringsAndTextProcessing/ReverseString/ReverseString.cs
//### Problem 2. Reverse string
//*	Write a program that reads a string, reverses it and prints the result at the console.

//_Example:_

//|  input | output |
//|:------:|:------:|
//| sample | elpmas |

using System;
using System.Linq;
using System.Text;

class ReverseString
{
    static void Main()
    {
        Console.Write("Enter a string: ");
        string input = Console.ReadLine();

        StringBuilder reversed = new StringBuilder();

        for (int i = input.Length - 1; i >= 0 ; i--)
        {
            reversed.Append(input[i]);
        }

        Console.WriteLine("Reversed: " + reversed);
    }
}

File: Javascript-Basics/Strings/ReverseString.js
/*
 /*### Problem 1. Reverse string
 *	Write a JavaScript function that reverses a string and returns it.
 _Example:_
 |  input | output |
 |:------:|:------:|
 | sample | elpmas |*/

function reverseString(string) {
    var reverse = '';
    for (var i = string.length - 1; i >= 0; i--) {
        reverse += string[i];
    }
    return console.log(reverse);
}
reverseString('Pesho');